tests/kubernetes/test_pod_generator.py

```diff
@@ -365,6 +365,7 @@ def test_reconcile_pods(self, mock_uuid):
                         image='',
                         name='name',
                         command=['/bin/command2.sh', 'arg2'],
+                        env=[],
```

Comment:
Is this change actually needed? It feels like it shouldn't be required, and the code needs to deal with not env not being specified if it doesn't.

```suggestion
```